ush/global_ens/evs_get_gens_atmos_data.sh

```diff
@@ -254,13 +275,19 @@ if [ $modnam = cmce ] ; then
           cat $WORK/grabcmce.${ihour}.${mb}.${h3} >> $WORK/cmce.sfc.${ihour}.${mb}.${h3}
           $WGRIB2 $cmce|grep "CAPE:atmos col"|$WGRIB2 -i $cmce -grib $WORK/grabcmce.${ihour}.${mb}.${h3}
           cat $WORK/grabcmce.${ihour}.${mb}.${h3} >> $WORK/cmce.sfc.${ihour}.${mb}.${h3}
+	  #****************************************************************************************
+	  #Note:  SPFH is still not available. So close this one
           #$WGRIB2 $cmce|grep "SPFH:"|$WGRIB2 -i $cmce -grib $WORK/grabcmce.${ihour}.${mb}.${h3}
           #cat $WORK/grabcmce.${ihour}.${mb}.${h3} >> $WORK/cmce.sfc.${ihour}.${mb}.${h3}
           #echo "cmce.upper" | $EXECevs_g2g/evs_g2g_adjustCMC.x
           #cat $WORK/cmce.sfc >> $WORK/cmce.upper.adjusted
+          *****************************************************************************************
```

Comment:
@BinbinZhou-NOAA On Line 284 in [ush/global_ens/evs_get_gens_atmos_data.sh](https://github.com/NOAA-EMC/EVS/pull/357/files#diff-13db435c6e9687549bffbc9208b764c545c0309a7f983bb6fa7f1f7237c9d3cf), please put a # before the ******************* so that the code knows it's a comment!

@ShelleyMelchior-NOAA @PerryShafran-NOAA